examples/navier-stokes/densitycurrent.h

```diff
@@ -54,7 +54,7 @@
 //
 // Conversion to Conserved Variables:
 //   rho = P0 Pi**(cv/Rd) / (Rd theta)
-//   E   = rho (cv theta Pi + (u u)/2 )
+//   E   = rho (cv theta Pi + (u u)/2 + g z)
```

Comment:
I would homogenize the convention chosen here with the one on line 156. I know that `theta Pi = T`, but just for consistency sake I would write `E   = rho (cv T + (u u)/2 + g z)` both here and on line 156.